packages/styles/scss/components/lightbox-media-viewer/_lightbox-media-viewer.scss

```diff
@@ -136,6 +136,11 @@
         @include carbon--make-col-ready;
         @include carbon--make-col(12, 16);
 
+        width: auto;
+        flex: none;
```

Comment:
This is constraining video media width in Safari.

<img width="1918" alt="Screen Shot 2022-10-13 at 1 00 05 PM" src="https://user-images.githubusercontent.com/12532727/195659809-9690d8ca-98f0-40dd-b8fc-87e22511fde0.png">

Reply:
I can't replicate this issue in Desktop Safari (version 16.2). It behvaes the same as Chrome. Tested at this link: https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/9149/iframe.html?id=components-lightbox-media-viewer--with-carousel&viewMode=story

Is this still reproducible for you @jkaeser ?

Reply:
@m4olivei Nope, I can no longer reproduce. Checked on Safari 16.1.

Reply:
Hey @jkaeser looks like the review is needing your approval from this review you left a ways back.